packages/@okta/vuepress-site/docs/guides/registration-inline-hook/add-registration-code/nodejs/get-submitted-credentials.md

```diff
@@ -0,0 +1,13 @@
+The following code represents a simple check on the email property of an incoming request from Okta; that is, the email used to self-register. The values of `data.userProfile` from within the request body contains the email property that will be verified. The variable `emailRegistration` stores the value of the email.
+
+
+
+```javascript
+app.post('/registrationHook', async (request, response) => {
+  var emailRegistration = request.body.data.userProfile['email'];
+
+
+}
+```
+
+>**Note:** The method definition begun in this code snippet is incomplete and is completed in the <GuideLink link="../send-response-code/">Send response code</GuideLink> section.
```

Comment:
Could you please add as a reviewer the person you worked with on the code snippets.